### PR TITLE
Seek Time Adjustment

### PR DESCRIPTION
**Pull request by CoolLeRue**
Added mapping and action to change the seek time using fn + </> when in Player mode.

Parameter added to the sampler settings called SEEK_TIME_JUMP. 
This parameter sets how much the seek time is adjusted by each time fn + </> is pressed. SEEK_TIME_JUMP also stores the custom seek time.

### CODE REVIEW

**Comment by langolierz** [comment]
looks really nice ! thanks for contributing ! this is the first external pull request so pretty excited :) , i will just want to test it out myself first, but also just another thought if i understand correctly:

the way this is set up we have three different related variables in the equation:
- the __player_position__ ; where the player is at
- the __seek_time__ ; how much is added to _player_position_ when i press `>`
- the __seek_time_jump__ ; how much is added to _seek_time_ when i press `fn + >`

i wonder if a similar amount of control over the seeking could be achieved by just cycling through the __seek_time__ amounts either in the setting or through the `fn + >` mapping ?

for example if the seek_time setting held [30, 60, 120, 240, 580], which could be cycled through in the setting mode, _or_ cycled through by pressing `fn + >` and `fn + <`

i like the way you have done it, and it does give you more control , but also thinking about balance of complexity vs control for people to understand how things work, what do you think ?

**Comment by CoolLeRue** [comment]
Check it out now. Took a while for me to figure out how to iterate through ['options'] but got there. Then it was just a matter of adding breaks to my if/elif checks in my loops... :D

**Comment by langolierz** [comment]
cool , yeah. and a big part of a customisable tool is you can set it up exactly how _you_  want it ! i find the controls on recur incredibly intuitive and can navigate the sampler while performing live but thats because they are exactly how i wrote them and an extension of what seemed logical to me. its quite hard to imagine what they seem like to someone new to it... and the act of writing your own controls means you are gonna be more familiar with it

i havent really figured out yet how to distinguish between personal mods (which i encourage anyone who has an interest in to do) and mods that would benefit all users ... 
i guess that will clarify over time and with feedback etc from more users

also how to allow receiving updates without whitewashing everything custom you have made for you (prob using git merge tools. although that can get messy !)

**Comment by CoolLeRue** [comment]
Yeah. I've been keeping a separate backup in case anything gets rolled over. FWIW I think your idea for implementing this is much cleaner than my first pass. Much more nimble to control. Also you really don't need a super specific seek range, just a couple small options and a couple broad options.

File: actions.py [hunk 1]
```diff
@@ -112,11 +112,33 @@ def toggle_show_on_player(self):
         elif self.data.player_mode == 'next':
             self.video_driver.next_player.toggle_show()
 
+    def increase_seek_time(self):
+        for idx, i in enumerate(self.data.settings['sampler']['SEEK_TIME']['options']):
```
Comment (langolierz) [inline actions.py]: this looks good! let me make a suggestion in the interest of educating (and coz im excited to be collaborating after many solo sessions wishing someone would be questioning what im writing!)

have you come across the % (modulo) function before ? it is a good way to 'wrap around' like what your if/elif statement is doing, but with the added advantage that you do not need to know (hardcode) the size of the list you are wrapping around, so if i was to for example add another step size to my settings code i wouldnt also need to update this function,

perhaps something like `new_index = (current_index + 1) % length_of_list`

**Comment by langolierz** [comment]
your pull request looks good - i just want to test it first but not finding the time while travelling atm , in meantime i put in a (untested change) of how i was thinking of doing it with modulo - its all the same in the end and i dont mind how it goes in just wanted to show you if you are interested

**Pull request merged**